account_payment_batch_process/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Batch Payments Processing",
+    "summary": "Process Payments in Batch",
+    "version": "11.0.1.0.1",
```

Comment:
```suggestion
    "version": "12.0.1.0.0",
```

Reply:
Why `12.0.1.0.0`? the code is migrated based on V11.0